test/cli/proj2_test.py

```diff
@@ -73,6 +73,17 @@ def test_absolute_path():
     assert stdout.find('Checking %s ...' % file1) >= 0
     assert stdout.find('Checking %s ...' % file2) >= 0
 
+def test_progress_threads():
+    size1 = os.path.getsize(os.path.join(__proj_dir, 'a', 'a.c'))
+    size2 = os.path.getsize(os.path.join(__proj_dir, 'b', 'b.c'))
+    perc1 = 100 * size1 // (size1 + size2)
+    perc2 = 100 * size2 // (size1 + size2)
+    __create_compile_commands()
+    ret, stdout, _ = cppcheck(['--project=' + os.path.join(__proj_dir, __COMPILE_COMMANDS_JSON), '-j2'], cwd=__script_dir)
```

Comment:
Please drop the `-j2` so it will be tested with and without threads in the CI via `TEST_CPPCHECK_INJECT_J`.